lib/config/config-file.js

```diff
@@ -313,7 +313,7 @@ function write(config, filePath) {
 function getBaseDir(configFilePath) {
 
     // calculates the path of the project including ESLint as dependency
-    var projectPath = path.resolve(__dirname, "../../../");
+    var projectPath = path.resolve(__dirname, "../../../../");
```

Comment:
@mysticatea You might be right, in this case `basedir` doesn't look right. But with your fix, globally installed ESLint is failing even worse. It's now two directories off from where it should be looking (before this fix, it was one dir off).